llvm_util/llvm2alive.cpp

```diff
@@ -160,7 +160,7 @@ class llvm2alive_ : public llvm::InstVisitor<llvm2alive_, unique_ptr<Instr>> {
     case llvm::Instruction::FPToSI:   op = ConversionOp::FPToSInt; break;
     case llvm::Instruction::FPToUI:   op = ConversionOp::FPToUInt; break;
     case llvm::Instruction::PtrToInt: op = ConversionOp::Ptr2Int; break;
-    case llvm::Instruction::IntToPtr: op = ConversionOp::Int2Ptr; break;
+    case llvm::Instruction::IntToPtr:
```

Comment:
please remove this diff. It's better to have int2ptr return an invalid expression that skipping a whole function.
Also, this is unrelated with the rest of the PR.

Reply:
Hmm, I'm still a fan of printing Unsupported operation when meeting inttoptr, but anyway it is not related with this patch, so reverted